app/models/setting/discovered.rb

```diff
@@ -11,6 +11,7 @@ def self.load_defaults
       [
         self.set('discovery_fact', _("The default fact name to use for the MAC of the system"), "discovery_bootif"),
         self.set('discovery_auto', _("Automatically provision newly discovered hosts, according to the provisioning rules"), false),
+        self.set('discovery_reboot', _("Automatically reboot discovered host during provisioning"), true),
```

Comment:
Use `N_()` and not `_` as this is an initialisation act, we translate them properly later.  The above lines are also incorrect.